### ISSUE TITLE
Odoo (Helm) Rancher 2 Install Not Working

### ISSUE THREAD

**Issue by gridworkz** [bitnami/bitnami-docker-odoo#52]
Hi: 
I can't install Odoo (Helm) from Rancher catalog. Rancher 2.1.1. I've tried a number parameter combinations (and no parameters). I always get 1 initializing and 1 unavailable and it stays there.
I've got cert-manager, nnginx-ingres and OpenEBS setup fine - Adding Odoo just doesn't seem to work no matter what I try. I'm using the latest Helm install template 5.0.

Are there any examples of this in a running state? If so, how was it accomplished?

Cheers, Dave

**Comment by tompizmor** [comment]
Hi @gridworkz,

We have not tried to install the Odoo chart using Rancher so we don't have any specific documentation about this. 

Can you provide more info about the error? Do you have access to the unavailable pod logs?

**Comment by gridworkz** [comment]
Hi Tomas:

It's working now. Thanks for getting back.

Cheers,
Dave

**Comment by tompizmor** [comment]
I am glad to hear that!

Do you know what was the issue? Just in case someone else finds the same problem than you.

**Comment by gridworkz** [comment]
Hi Tomas:

Ah yes, sorry I replied to Ricky on that. There was a PVC still present
from a previous install. I just removed it via the Rancher UI. (under
storage -> persistent volumes). Sometimes it says 'released' but I delete
it anyway.

Cheers,
Dave

**Comment by gridworkz** [comment]
Hi Tomas:

I forgot I also added a Postgres password to the list of parameters I sent
down before launching Odoo on Rancher. Quick question - how do folks
usually handle the Odoo addons folder in Kubernetes? Essentially I'm trying
to setup a shared directory where the addons reside in one spot. Then all
the Odoo instances can use it.

I have spare cycles and I can help you guys with scaling Odoo on K8s /
Rancher. We've decided on the Bitnami Odoo Helm charts for our service so
I'm at your service.

Cheers,
Dave

**Comment by tompizmor** [comment]
OK! Thanks for writing back.

> how do folks usually handle the Odoo addons folder in Kubernetes?

That sounds to me like you would need an NFS volume shared between all your Odoo installations but I don't know if there are any other possibilities. 

> I have spare cycles and I can help you guys with scaling Odoo on K8s /
Rancher. We've decided on the Bitnami Odoo Helm charts for our service so
I'm at your service.

Thanks! Any help is more than welcome! Feel free to check the helm/chart repo for issues and PR affecting Odoo.

**Comment by gridworkz** [comment]
Hi Tomas:

Quick observation/question. I just setup a K8 cluster for a client. They
wanted to use RancherOS on all nodes. Everything went fine except the Odoo
Helm chart had a little hiccup. It doesn't seem to issue internal ips to
the Odoo/Postgres pods and remains in "initialization" state. I confirmed
that open-iscsi was installed/up on each node. I also installed
Cert-manager and OpenEBS and they seem to issue ips to their respective
pods. So, my question - is Bitnami Odoo Helm chart compatible with
RancherOS? If anyone has tried it - are there any tricks? Any pointers
appreciated - also willing to work on this if there's glitches....

Cheers,
Dave
